docs/MultiViewOpenEXR.rst

```diff
@@ -44,17 +42,138 @@ Stereo viewing software may default to assuming that it should send the
+This is usually the part that contains the R,G,B channels. Subsequent parts can be in any order.
+Software that only reads the first part of a file will therefore read the intended channels from the intended view
+
+Part Naming
+-----------
+OpenEXR requires a unique ``name`` attribute in each part, to allow consistent indexing of parts. It is convenient
+to include the view and layer name within the part name, but software reading the file should not make
+assumptions about ``name`` attribute. Instead, the view name should be read from the ``view`` attribute and the
+layer name for each channel derived from the channel name.
+
+
+Channel Naming
+--------------
+The channel names should not include the view name.
+Thus, a stereo RGB file will contain two ``R`` channels,
+one for left and one for right, in two separate parts.
+
+
+Channels Not in a View
+----------------------
+In some cases, channels may not be associated with a view. Such channels should be stored in a part without
+a ``view`` attribute. Since the first part specifies the default view, this part cannot be the first part
+within the file
+
+
+Example File Structures
+-----------------------
+
+Simple stereo EXR, with default view left::
+
+  Part 0: view: "left" ; name: "rgbleft"
+          channels: R G B
+
+  Part 1: view: "right" ; name: "rgbright"
+          channels: R G B
+
+Simple stereo EXR, with default view right::
+
+  Part 0: view: "right" ; name: "rightrgb"
+          channels: R G B
+
+  Part 1: view: "left" ; name: "leftrgb"
+          channels: R G B
+
+Stereo EXR containing diffuse and specular layers (two parts), default view right::
+
+  Part 0: view: "right" ; name: "right"
+          channels: R G B diff.R diff.G diff.B spec.R spec.G spec.B
+
+  Part 1: view: "left" ; name: "left"
+          channels: R G B diff.R diff.G diff.B spec.R spec.G spec.B
+
+Stereo EXR containing diffuse and specular layers (six parts), default view right::
+
+  Part 0: view: "right" ; name: "rgb-right"
+          channels: R G B
+
+  Part 1: view: "left" ; name: "rgb-left"
+          channels: R G B
+
+  Part 2: view: "left" ; name: "diff-left"
+          channels: diff.R diff.G diff.B
+
+  Part 3: view: "right" ; name: "diff-right"
+          channels: diff.R diff.G diff.B
+
+  Part 4: view: "left" ; name "spec-left"
+          channels: spec.R spec.G spec.B
+
+  Part 5: view: "right" ; name "spec-right"
+          channels: spec.R spec.G spec.B
+
+Stereo EXR containing non-view part::
+
+  Part 0: view: "right" ; name: "rightrgb"
+          channels R G B
+
+  Part 1: view: "left" ; name: "leftrgb"
+          channels R G B
+
+  Part 2: name: "annotations"
+          channels notes.R notes.G notes.B
+
+
+Storing Multi-View Images in Single Part OpenEXR files
+======================================================
+
+A single part multi-view OpenEXR file is identified by the presence of a
```

Comment:
All through this doc, we talk of multi-view, multi-part, multiView. My preference would be to just conform to the feature name, multiView, to reduce having to think about terminology and which one is found in a file, versus in documentation. It would reduce the number of search terms used if someone is trying to find all the references.